src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1868,6 +1868,17 @@ internal enum ErrorCode
         ERR_CopyConstructorWrongAccessibility = 8878,
         ERR_NonPrivateAPIInRecord = 8879,
 
+        // The following warnings correspond to errors of the same name, but are reported
+        // when a definite assignment issue is reported due to private fields imported from metadata.
+        WRN_UnassignedThisAutoProperty = 8880,
```

Comment:
Do we want to reserve a block of warnings for warning wave warnings?

Reply:
I don't think so.

---
In reply to: [463269817](https://github.com/dotnet/roslyn/pull/46221#discussion_r463269817) [](ancestors = 463269817)